hbase-common/src/test/java/org/apache/hadoop/hbase/HBaseClassTestRule.java

```diff
@@ -82,6 +99,49 @@ private static long getTimeoutInSeconds(Class<?> clazz) {
         clazz.getName() + " does not have SmallTests/MediumTests/LargeTests in @Category");
   }
 
+  /**
+   * @param clazz Test class that is running.
+   * @return the number of parameters for this given test class. If the test is not parameterized or
+   *   if there is any issue determining the number of parameters, returns 1.
+   */
+  private static int getNumParameters(Class<?> clazz) {
+    RunWith[] runWiths = clazz.getAnnotationsByType(RunWith.class);
+    boolean isTestParameterized = runWiths != null && Arrays.stream(runWiths).anyMatch(
```

Comment:
FYI, isTestParameterized is name of a method, not a variable. testParameterized would be the variable name.